### ISSUE TITLE
Orchid reports itself as a SNAPSHOT version

### ISSUE THREAD

**Issue by bjansen** [orchidhq/Orchid#365]
The logs show:

```
[INFO] Orchid: Running Orchid version 0.20.0-SNAPSHOT, site version 0.1 in debug environment
```

And `OrchidVersion` v0.20.0 contains:

```java
    public static String getVersion() {
        return "0.20.0-SNAPSHOT";
    }
```

**Comment by cjbrooks12** [comment]
Ah, good catch. That number is only ever used for display so it shouldn't impact anything in 0.20.0, but I'll make sure it's fixed in the next release.

**Comment by cjbrooks12** [comment]
This is now fixed and released in 0.21.0
